src/components/Menu/keys.js

```diff
@@ -0,0 +1,24 @@
+const isMac = (navigator.userAgent.includes('Mac'))
+
+const MODIFIERS = {
+	Mod: isMac ? 'Meta' : 'Control',
+	Alt: 'Alt', // Option key, on Apple computers.
+	Control: 'Control',
+	Shift: 'Shift',
+
+	// unused
+	// AltGraph: 'AltGraph',
+	// Meta: 'Meta', // Command key on Apple computers
+}
+
+const TRANSLATIONS = {
+	[MODIFIERS.Mod]: t('text', isMac ? 'Command' : 'Control'),
```

Comment:
```suggestion
	[MODIFIERS.Mod]: isMac ? t('text', 'Command') : t('text', 'Control'),
```

Ah, I missed that, we need two separate `t()` calls so that the translation script is picking this up properly picking that up.